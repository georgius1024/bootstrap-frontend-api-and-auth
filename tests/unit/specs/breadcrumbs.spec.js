import {
  shallowMount
} from '@vue/test-utils'

import component from '@/components/app/breadcrumbs'

describe('breadcrumbs.vue', () => {
  const wrapper = shallowMount(component)
  it('should render correct contents', () => {
    expect(wrapper).toMatchSnapshot()
  })
  it('should show items array', () => {
    wrapper.setProps({ items: [
      { text: 'one' },
      { text: 'two' }
    ] })
    expect(wrapper).toMatchSnapshot()
  })
})
