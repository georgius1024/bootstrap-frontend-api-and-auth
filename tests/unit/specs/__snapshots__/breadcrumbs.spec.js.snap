// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breadcrumbs.vue should render correct contents 1`] = `<vbreadcrumbs-stub divider="/" items="[object Object]"></vbreadcrumbs-stub>`;

exports[`breadcrumbs.vue should show items array 1`] = `<vbreadcrumbs-stub divider="/" items="[object Object],[object Object]"></vbreadcrumbs-stub>`;
